docs/src/TapArea.doc.js

```diff
@@ -176,8 +174,8 @@ card(
   `}
     name="Full width and full height"
     defaultCode={`
-<Box color="white" display="flex" width={500} height={250}>
-  <Box column={6}>
+<Box color="olive" display="flex" width={500} height={250}>
+  <Box borderSize='sm' margin={3} column={6}>
```

Comment:
@AlbertCarreras could you use double quotes here and in the rest of the changes?

Reply:
@AlbertCarreras fyi updated these myself